build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-alpha2'
+        classpath 'com.android.tools.build:gradle:3.0.0-alpha3'
```

Comment:
oh, hey, how do I do that? I'm kinda new using github+AS. I have forked your repo, then I used my fork on AS to commit and push. And then I made the pull request using the github website. How do I update the project?